### ISSUE TITLE
Issues with end of line for Vue SFC blocks with CLRF/LF

### ISSUE THREAD

**Issue by alexrafael10** [meteorlxy/eslint-plugin-prettier-vue#13]
in the `processSFCBLock` the code is doing

```
const endingTag = `</${type}>\n`
```

The usage of `\n`  it's making repos not using LF show an unfixable error at the end of SFC `</script>` tag:

![image](https://user-images.githubusercontent.com/21692735/94182535-88542280-fe77-11ea-9135-d6566a4510a5.png)

It will never go away since the plugin is parsing the `\n` added when parsing the SFC.

I'm forking in order to not mess with my repo line ends.

If this is actually an issue and maybe no one wants to spend time on it I can take some time maybe to send a PR controlling which line break char is inserted reading from `endOfLine` prettier option.

Cheers.

Ps: your plugin saved me so much pain introduced by prettier and vue lol

**Comment by meteorlxy** [comment]
Ah yes, didn't notice that.

**Comment by meteorlxy** [comment]
Simply reading the `endOfLine` option from prettier config is not a perfect solution.

For example, it will cause a false negative:

```
<scirit>\r\n
// ...\r\n
</script>\n
```

In this case, the last `\n` will be replaced by `\r\n`, and won't be reported by prettier.

So respect the original eol char could be a better solution 🤔

**Comment by alexrafael10** [comment]
Yeah, i actually noticed that yesterday haha i came up with this:

```const endingTag = `</${type}>${content.includes('\r\n') ? '\r\n' : '\n'}```

My only worry for this is that to make it false it will need to parse the whole content in order to say "ok there is no `\r\n` in this file. (had to come up with a very quick and dirty solution for my work)

**Comment by alexrafael10** [comment]
This is how eslint guys achieve this> https://github.com/eslint/eslint/blob/master/lib/rules/linebreak-style.js

**Comment by machao** [comment]
any plan to fix it?
